src/lib/split-view/split-view/split-view.ts

```diff
@@ -0,0 +1,90 @@
+import { CustomElement, attachShadowTemplate, ICustomElement, FoundationProperty, coerceBoolean } from '@tylertech/forge-core';
+
+import { SplitViewAdapter } from './split-view-adapter';
+import { SplitViewFoundation } from './split-view-foundation';
+import { SplitViewOrientation, SPLIT_VIEW_CONSTANTS } from './split-view-constants';
+import { ISplitViewPanelComponent, SplitViewPanelComponent } from '../split-view-panel';
+import { ISplitViewBase } from '../core/split-view-base';
+
+import template from './split-view.html';
+import styles from './split-view.scss';
+
+export interface ISplitViewComponent extends ISplitViewBase, ICustomElement {
+  orientation: SplitViewOrientation;
+  stackSlottedPanels(target: ISplitViewPanelComponent): void;
+  unstackSlottedPanels(): void;
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'forge-split-view': ISplitViewComponent;
+  }
+}
+
+@CustomElement({
+  name: SPLIT_VIEW_CONSTANTS.elementName,
+  dependencies: [SplitViewPanelComponent]
+})
+export class SplitViewComponent extends HTMLElement implements ISplitViewComponent {
```

Comment:
Let's update this to extend `BaseComponent` from core to be consistent with other components (I know it doesn't technically share anything right now, but it's there for that purpose in the future).